crates/e2e/src/nodes/mod.rs

```diff
@@ -43,6 +43,8 @@ impl Node {
             "1",
             "--timestamp",
             "1577836800",
+            "--hardfork",
+            "cancun",
```

Comment:
Still does not need to be checked in...

Reply:
No worries, this won't be committed at all. It just makes my life easier while implementing it, I still need to solve this issue, didn't find time for it yet...